zebra/zebra_vty.c

```diff
@@ -1062,7 +1066,6 @@ static void vty_show_ip_route(struct vty *vty, struct route_node *rn,
 		show_route_nexthop_helper(vty, re, nexthop);
 		vty_out(vty, ", %s\n", up_str);
 	}
-
```

Comment:
Can we restore this blank line - was there some reason to delete it?

Reply:
Ok, will restore it.